app/org/sagebionetworks/bridge/models/schedules/ABTestScheduleStrategy.java

```diff
@@ -113,22 +114,11 @@ public void validate(Errors errors) {
                 return;
```

Comment:
I just noticed that this errors.popNestedPath() and return means the validator skips out on the first null schedule instead of validating all the schedules in the list.